src/main/java/org/jfrog/bamboo/utils/Utils.java

```diff
@@ -0,0 +1,73 @@
+package org.jfrog.bamboo.utils;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Utils {
+    /**
+     * Creates and configures an ObjectMapper instance.
+     *
+     * @return The configured ObjectMapper instance.
+     */
+    public static ObjectMapper createMapper() {
+        return new ObjectMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+                .disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
+                .setSerializationInclusion(JsonInclude.Include.NON_NULL);
+    }
+
+    /**
+     * Removes the surrounding quotes from the given string, if present.
+     *
+     * @param str The string to unquote.
+     * @return The unquoted string.
+     */
+    public static String unQuote(String str) {
+        if ((str.startsWith("\"") && str.endsWith("\"")) || (str.startsWith("'") && str.endsWith("'"))) {
+            return str.substring(1, str.length() - 1);
+        }
+        return str;
```

Comment:
I almost sure it does the same (please check me):
```suggestion
        return StringUtils.strip(str, "\"'");
```